lectures/intro_supply_demand.md

```diff
@@ -243,7 +240,7 @@ The difference between willingness to sell and the actual price is producer surp
 
 The value $S_p(q)$ is the integral of these surpluses.
 
-Evaluating the integral in the definition of consumer surplus gives
+Evaluating the integral in the definition of producer surplus {eq}`eq:pdcr_spls` gives
```

Comment:
@HengchengZhang this will resolve as number reference. Do you think it should be in `(` brackets?

Reply:
Do you mean the number reference should be in brackets? I think our theme automatically applied this like `(7.1)`.

Reply:
Oh that's great 👍